Source/WebCore/rendering/mac/RenderThemeMac.mm

```diff
@@ -750,39 +748,36 @@ static NSControlSize controlSizeFromPixelSize(const IntSize* sizes, const IntSiz
     return NSControlSizeMini;
 }
 
-static const int* popupButtonMargins(NSControlSize size)
+static std::span<const int, 4> popupButtonMargins(NSControlSize size)
 {
-    static const int margins[4][4] =
-    {
-        { 0, 3, 1, 3 },
-        { 0, 3, 2, 3 },
-        { 0, 1, 0, 1 },
-        { 0, 6, 2, 6 },
+    static constexpr std::array<std::array<int, 4>, 4> margins {
```

Comment:
I think it’s a little nicer to use auto and std::to_array, so you don’t have to write out the ugly nested std::array declaration, and if someone adds or removes an entry they don’t have to update the type declaration.